packages/presentation/lib/src/pages/login_page/widgets/error_message_widget.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/src/pages/login_page/bloc/login_bloc.dart';
+import 'package:presentation/style/dimens.dart';
+
+class ErrorMessageWidget extends StatelessWidget {
+  final LoginBloc bloc;
+  final String? tile;
+
+  const ErrorMessageWidget({
+    required this.bloc,
+    required this.tile,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    if (tile == null || tile!.isEmpty) return const SizedBox.shrink();
+    return Padding(
+      padding: EdgeInsets.only(bottom: Dimens.size20),
+      child: Text(
+        tile ?? '',
```

Comment:
create an extension orEmpty() plz and use it instead of ?? ''

Reply:
completely agree =)